src/shark.js

```diff
@@ -27,6 +27,8 @@ const ServiceSettingsClient = require('./clients/service/settings-client')
 const SubscriptionClient = require('./clients/subscription-client')
 const UserClient = require('./clients/user-client')
 
+const FeedbackPagesHelper = require('./helpers/feedback-pages-helper')
```

Comment:
I am still really unhappy with the name. 

I am not even sure, that this npm package is the correct place for this, but let's not think about this for now. But I'm going to raise that issue when you try to build the next `Helper`.

```
const FeedbackPages = require('./helpers/feedback-pages')
```
is better in my opinion.